vm/src/builtins/type.rs

```diff
@@ -271,7 +267,7 @@ impl PyType {
     fn dir(zelf: PyRef<Self>, vm: &VirtualMachine) -> PyList {
         let attributes: Vec<PyObjectRef> = zelf
             .get_attributes()
-            .drain()
+            .drain(..)
```

Comment:
I know `drain()` was already here, but this can actually just be `.into_iter()`.